src/test/java/uk/co/ramp/people/PopulationGeneratorTest.java

```diff
@@ -83,19 +91,62 @@ public void generate() {
     double women =
         population.values().stream().map(Case::gender).filter(a -> a.equals(Gender.FEMALE)).count()
             / (double) popSize;
+    double healthModifier = populationGenerator.getHealthModifier(50);
 
     double hasApp = population.values().stream().filter(Case::hasApp).count() / (double) popSize;
 
     System.out.println(hasApp);
 
     Assert.assertEquals(0.5, compliance, 0.01);
-    Assert.assertEquals(0.5, health, 0.01);
+    Assert.assertEquals(0.5 * healthModifier, health, 0.01);
     Assert.assertEquals(50, age, 0.5);
     Assert.assertEquals(0.5, men, 0.01);
     Assert.assertEquals(0.5, women, 0.01);
     Assert.assertEquals(0.7, hasApp, 0.01);
   }
 
+  @Test
+  public void getHealthModifier() throws FileNotFoundException {
+    PopulationProperties populationProperties = TestUtils.populationProperties();
+
+    double max =
+        populationProperties.populationAges().values().stream()
+            .mapToDouble(MinMax::max)
+            .max()
+            .orElse(100);
```

Comment:
Since we know what the populationProperties values are, I would prefer to have a simpler test, with `double max = 100;` here.  (I'd also prefer to call it `maxAge`.)